ui/apps/dashboard/src/lib/inngest/functions/run-network.ts

```diff
@@ -7,6 +7,15 @@ import { createChannel } from '../realtime';
 import { createInsightsNetwork } from './agents/network';
 import type { InsightsAgentState } from './agents/types';
 
+type PublishOpts = {
+  topics: string[];
+  channel: string;
+  runId: string;
+};
+const publishApi = async ({ topics, channel, runId }: PublishOpts, data: any) => {
+  return await inngest['inngestApi'].publish({ topics, channel, runId }, data);
+};
```

Comment:
This shouldn't be necessary when this lands: https://github.com/inngest/inngest-js/pull/1093